src/packet.rs

```diff
@@ -37,15 +37,15 @@ use crate::ranges;
 use crate::recovery;
 use crate::stream;
 
+use crate::MAX_CONN_ID_LEN as MAX_CID_LEN;
```

Comment:
Since this is only used in 3 places, it's probably better to just inline `crate::MAX_CONN_ID_LEN` in those places.

Reply:
Yeah but we still need a separate constant with `u8` type to pass typecheck. I'll adjust code a bit.